src/packages/client/src/generation/TSClient.ts

```diff
@@ -477,12 +502,154 @@ class PrismaClientClass implements Generatable {
  * 
  * Read more in our [docs](https://www.prisma.io/docs/reference/tools-and-interfaces/prisma-client).
  */`
+  }
+  public toTSWithoutNamespace(): string {
+    const { dmmf } = this
+    return `${this.jsDoc}
+export class PrismaClient<
+  T extends Prisma.PrismaClientOptions = Prisma.PrismaClientOptions,
+  U = 'log' extends keyof T ? T['log'] extends Array<Prisma.LogLevel | Prisma.LogDefinition> ? Prisma.GetEvents<T['log']> : never : never
+      > {
+      /**
+       * @private
+       */
+      private fetcher;
+      /**
+       * @private
+       */
+      private readonly dmmf;
+      /**
+       * @private
+       */
+      private connectionPromise?;
+      /**
+       * @private
+       */
+      private disconnectionPromise?;
+      /**
+       * @private
+       */
+      private readonly engineConfig;
+      /**
+       * @private
+       */
+      private readonly measurePerformance;
+
+      ${indent(this.jsDoc, tab)}
+    constructor(optionsArg ?: T);
+    $on<V extends U>(eventType: V, callback: (event: V extends 'query' ? Prisma.QueryEvent : Prisma.LogEvent) => void): void;
+    /**
+     * @deprecated renamed to \`$on\`
+     */
+    on<V extends U>(eventType: V, callback: (event: V extends 'query' ? Prisma.QueryEvent : Prisma.LogEvent) => void): void;
+    /**
+     * Connect with the database
+     */
+    $connect(): Promise<void>;
+    /**
+     * @deprecated renamed to \`$connect\`
+     */
+    connect(): Promise<void>;
```

Comment:
I thought deprecated methods were removed, maybe need to update it in this PR too?